src/method/method_def.cpp

```diff
@@ -223,48 +255,60 @@ method_sig_node::operator= (const method_sig_node &obj)
 	  strncpy (method_name, obj.method_name, method_name_len);
 	}
 
+      if (obj.auth_name != nullptr)
+	{
+	  int auth_name_len = strlen (obj.auth_name);
+	  auth_name = (char *) db_private_alloc (NULL, auth_name_len + 1);
```

Comment:
In this source code, NULL check is not performed after allocating the urinary tract with db_private_alloc().
Is there any need to check it?

Reply:
The code is inside of `operator=`. I also have been thinking about NULL checking, but exception handling of C++ objects is not yet well supported in CUBRID code. This is an issue that needs to be considered through refactoring in the future.

Reply:
OK, I agree.